native/src/macros.rs

```diff
@@ -82,13 +91,13 @@ macro_rules! cast_to_slice {
     }};
 }
 
-macro_rules! cast_to_string {
-    ($cx:expr, $obj:expr) => {{
-        $obj.downcast::<JsString>()
-            .unwrap_or($cx.string(""))
-            .value()
-    }};
-}
+// macro_rules! cast_to_string {
```

Comment:
Should we delete this rather than comment it out? Same for line 58 - 65 above as well.

Reply:
+1 please remove